src/components/dialog/demoBasicUsage/script.js

```diff
@@ -4,18 +4,26 @@ angular.module('dialogDemo1', ['ngMaterial'])
   $scope.alert = '';
 
   $scope.showAlert = function(ev) {
+    // Appending dialog to document.body to cover sidenav in docs app
+    // Modal dialogs should fully cover application
+    // to prevent interaction outside of dialog
     $mdDialog.show(
-      $mdDialog.alert()
+      $mdDialog.alert({
+          parent: angular.element(document.body)
```

Comment:
@marcysutton - I just added support for `.parent(angular.element(document.body))` in master.
Also I think you should be using ` $rootElement.find('body') || $rootElement` instead of `document.body`